Demo/Application/Features/VenmoViewController.swift

```diff
@@ -66,7 +66,7 @@ class VenmoViewController: PaymentButtonBaseViewController {
                 progressBlock("Got a nonce 💎!")
                 completionBlock(venmoAccount)
             } catch {
-                if (error as NSError).code == 10 {
+                if error as? BTVenmoError == .canceled {
```

Comment:
🌟

Reply:
This is slick!

Reply:
🏈 Punt, but curious your thoughts.

For V7, we could get to something like this `if error == .canceled { ... }` if the error type we return is BT SDK specific. Is that what you're thinking as the V7 goal?

Reply:
Yeah - I think that would be dope if we return strongly typed errors. The only case it _could_ be a little weird is we would need to wrap any errors from core. I think in most cases we do that anyways but I know there are a few instances where we return a core error directly. Though I don't think it should be a blocker for strongly typed errors.